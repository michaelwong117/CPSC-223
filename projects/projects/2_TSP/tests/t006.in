19
BWI DCA IAD BDL HER TXL AMS ZRH SFO LAX SEA HNL MDW IAH CLT GSO RDU CDG HVN
39.175278 -76.668333
38.851944 -77.037778
38.947500 -77.460000
41.939167 -72.683333
35.339722 25.180278
52.559722 13.287778
52.308611 4.763889
47.463333 8.550000
37.618889 -122.375000
33.942500 -118.408056
47.450000 -122.311667
21.318611 -157.922500
41.785833 -87.752500
29.984444 -95.341389
35.213611 -80.949167
36.101389 -79.941111
35.877778 -78.787500
49.012778 2.550000
41.263889 -72.886944
