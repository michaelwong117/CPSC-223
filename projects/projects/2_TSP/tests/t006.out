-exchange adjacent:     47520.52 BWI DCA IAD BDL TXL HER ZRH AMS LAX SFO HNL SEA IAH MDW CLT GSO RDU CDG HVN BWI
